routes/userRoutes.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+const User = mongoose.model('user_collection');
+
+module.exports = (app) => {
+
+    app.get('/api/user/', async (req, res) => {
+        let users = await User.find(function (mail){mail.email && mail.name});
```

Comment:
was genau möchtest du hier machen? also gefiltert werden soll ja nach email und passwort. die werte dafür bekommst du aus dem request (`req`) und auf sie zugreifen kannst du mit `req.query.email` und `req.query.passwort`

außerdem ist hier ein link dazu wie man die DB mit mehreren bedindungen/ variablen durchsucht: https://docs.mongodb.com/manual/reference/method/db.collection.find/#query-for-multiple-conditions